DSharpPlus/Entities/Interaction/DiscordInteraction.cs

```diff
@@ -102,6 +102,18 @@ public DiscordChannel Channel
         [JsonProperty("message")]
         internal DiscordMessage Message { get; set; }
 
+        /// <summary>
+        /// Gets the locale of the user that invoked this interaction.
+        /// </summary>
+        [JsonProperty("locale")]
+        public string? Locale { get; set; }
```

Comment:
Also forgot to mention that public properties should not be mutable by users; this should be `internal set`, oops.

Reply:
As well as the other one, ofc.

Reply:
Fixed.